Harvest.Web/Controllers/Api/InvoiceController.cs

```diff
@@ -59,7 +59,7 @@ public async Task<ActionResult> Get(int projectId, int invoiceId)
         public async Task<ActionResult> List(int projectId, int? maxRows)
         {
             var user = await _userService.GetCurrentUser();
-            var hasAccess = await _userService.HasAccess(AccessCodes.FieldManagerAccess);
+            var hasAccess = await _userService.HasAccess(AccessCodes.FieldManagerAccess) || await _userService.HasAccess(AccessCodes.FinanceAccess);
```

Comment:
2 things:
1. This doesn't check for team roles, right?
2. Would be nice to extend or overload the HasAccess to check for an array of roles instead of using `||` because this will cause the user's roles to be queried from the Db multiple times, which isn't necessary

Reply:
I was thinking about the array of roles...
FYI, it calls this which checks the team:
https://github.com/ucdavis/Harvest/blob/main/Harvest.Core/Services/UserService.cs#L93

Reply:
@jSylvestre That doesn't check that team as far as I see.  It does return the team name/slug but doesn't check it anywhere.